### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by gzussa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+## datadog-api-client-1.0.0-beta.4 / 2020-06-09
+
+* [Added] Add Logs Archives endpoints. See [#333](https://github.com/DataDog/datadog-api-client-java/pull/333).
+* [Changed] Add missing values to enums. See [#327](https://github.com/DataDog/datadog-api-client-java/pull/327).
```
Comment (nmuesch) [inline CHANGELOG.md:6]: Can we use `[BREAKING]` for anything with backwards incompatibility (to be consistent with the previous releases)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+## datadog-api-client-1.0.0-beta.4 / 2020-06-09
+
+* [Added] Add Logs Archives endpoints. See [#333](https://github.com/DataDog/datadog-api-client-java/pull/333).
```
Comment (gzussa) [inline CHANGELOG.md:5]:
```suggestion
* [BREAKING] Add Logs Archives endpoints. See [#333](https://github.com/DataDog/datadog-api-client-java/pull/333).
```

File: CHANGELOG.md (hunk 1 above)
Comment (gzussa) [inline CHANGELOG.md:6]:
```suggestion
* [BREAKING] Add missing values to enums. See [#327](https://github.com/DataDog/datadog-api-client-java/pull/327).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+* [Changed] Add missing values to enums. See [#327](https://github.com/DataDog/datadog-api-client-java/pull/327).
+    * This change includes backwards incompatible changes when using the `MonitorSummary` widget.
+* [Changed] Split schemas from DashboardList v2. See [#323](https://github.com/DataDog/datadog-api-client-java/pull/323).
```
Comment (gzussa) [inline CHANGELOG.md:8]:
```suggestion
* [BREAKING] Split schemas from DashboardList v2. See [#323](https://github.com/DataDog/datadog-api-client-java/pull/323).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+    * This change includes backwards incompatible changes when using corresponding endpoints methods.
+* [Changed] Clean synthetics test CRUD endpoints. See [#322](https://github.com/DataDog/datadog-api-client-java/pull/322).
```
Comment (gzussa) [inline CHANGELOG.md:10]:
```suggestion
* [BREAKING] Clean synthetics test CRUD endpoints. See [#322](https://github.com/DataDog/datadog-api-client-java/pull/322).
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+    * This change includes backwards incompatible changes when using corresponding endpoints methods.
```
Comment (gzussa) [inline CHANGELOG.md:11]:
```suggestion
    * This change includes backwards incompatible changes when using corresponding endpoints methods.
* [Added] Add Logs Archives endpoints. See [#333](https://github.com/DataDog/datadog-api-client-java/pull/333).    
```

File: CHANGELOG.md (hunk 2 above)
Comment (gzussa) [inline CHANGELOG.md:5]:
```suggestion
```

**Pull request merged**